features/minion/heartbeat/consumer/src/main/java/org/opennms/minion/heartbeat/consumer/HeartbeatConsumer.java

```diff
@@ -216,16 +221,30 @@ private void provision(final OnmsMinion minion,
             return;
         }
 
+        final String prevForeignSource = String.format(PROVISIONING_FOREIGN_SOURCE_PATTERN, prevLocation);
+        final String nextForeignSource = String.format(PROVISIONING_FOREIGN_SOURCE_PATTERN, nextLocation);
+
+        PluginConfig policy = new PluginConfig(DEFAULT_SNMP_POLICY, "org.opennms.netmgt.provision.persist.policies.MatchingIpInterfacePolicy");
+        policy.addParameter("ipAddress", "~^docker.*$");
+        policy.addParameter("action", "DO_NOT_PERSIST");
```

Comment:
The IPAddress will never match docker.* here. I think you want to use the MatchingSnmpInterfacePolicy here, right?

Reply:
An example here: https://opennms.discourse.group/t/ignoring-docker-volumes-and-network-interfaces/1189

Reply:
> An example here: https://opennms.discourse.group/t/ignoring-docker-volumes-and-network-interfaces/1189

Thanks, bookmarked

Reply:
> The IP-Address will never match docker.* here. I think you want to use the MatchingSnmpInterfacePolicy here and match against ifDescr, right?

Thanks, I didn't know and couldn't find any documentation about it. Now I have some...